doc/getting_started/up2.rst

```diff
@@ -43,159 +44,155 @@ the recommended options for choosing an Automatic installation type,
 and using the UP2’s storage as the target device for installation
 (overwriting the existing data and creating three partitions on the
 UP2’s eMMC drive).
-After installation is complete, boot into Clear Linux, login as **root**,
+After the installation is complete, boot into Clear Linux, log in as **root**,
 and set a password.
 Clear Linux is set to automatically update itself. We recommend that
 you disable this feature to have more control over when the updates
 happen.
 
 Use this command to disable the autoupdate feature:
 
-.. code-block:: none
+.. code-block:: console
 
-      # swupd autoupdate --disable
+   # swupd autoupdate --disable
 
 Use the swupd bundle-add command and add these Clear Linux bundles:
 
-.. code-block:: none
+.. code-block:: console
 
-      # swupd bundle-add vim network-basic service-os kernel-pk os-clr-on-clr
+   # swupd bundle-add vim network-basic service-os kernel-pk os-clr-on-clr
 
 Building and installing software
---------------------------------
+================================
 
 Downloading ACRN source code
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+----------------------------
 
 .. code-block:: none
 
-      # git clone https://github.com/projectacrn/acrn-hypervisor
-      # cd acrn-hypervisor
+   # git clone https://github.com/projectacrn/acrn-hypervisor
+   # cd acrn-hypervisor
 
 Building ACRN hypervisor and the device model
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+---------------------------------------------
 
-.. code-block:: none
+We start first by
```

Comment:
Since there's no second step listed here, simply say:

    Build the hypervisor and device model images: